js/background.js

```diff
@@ -85,3 +85,17 @@ chrome.omnibox.onInputEntered.addListener( function(text) {
         chrome.tabs.update(tabs[0].id, {url: "https://duckduckgo.com/?q="+encodeURIComponent(text)});
     });
 });
+
+//This adds Context Menu when user select some text.
+//create context menu
+chrome.contextMenus.create({
+    title : 'Search anonymously on DuckDuckGo',
```

Comment:
@nirajkaushal would it be possible to use the `%s` placeholder here as described in [the docs](https://developer.chrome.com/extensions/contextMenus#property-create-createProperties)? That way we can use the same `Search DuckDuckGo for "%s"` here as we do in Firefox for instance

![](https://duckduckgo.com/extensions/thanks/images/context-search.png)